tasks/extract_filepaths_for_granule/test/unit_tests/test_extract_file_paths_for_granule.py

```diff
@@ -238,152 +238,16 @@ def test_task(self, mock_debug: MagicMock):
             extract_filepaths_for_granule.OUTPUT_DESTINATION_BUCKET_KEY: "sndbx-cumulus-public",
         }
         exp_gran = {
-            "dataType": "MOD09GQ_test-jk2-IngestGranuleSuccess-1558420117156",
-            "files": [
-                {
-                    "bucket": "cumulus-test-sandbox-protected",
-                    "duplicate_found": "True",
-                    "fileName": "MOD09GQ.A0219114.N5aUCG.006.0656338553321.h5",
-                    "key": "MOD09GQ___006/2017/MOD/MOD09GQ.A0219114.N5aUCG.006.0656338553321.h5",
-                    "path": "jk2-IngestGranuleSuccess-1558420117156-test-data/files",
-                    "size": 1098034,
-                    "type": "data",
-                    "url_path": "{cmrMetadata.Granule.Collection.ShortName}___"
-                                "{cmrMetadata.Granule.Collection.VersionId}/"
-                                "{extractYear(cmrMetadata.Granule.Temporal."
-                                "RangeDateTime.BeginningDateTime)}/"
-                                "{substring(file.name, 0, 3)}",
-                },
-                {
-                    "bucket": "cumulus-test-sandbox-private",
-                    "duplicate_found": "True",
-                    "fileName": "MOD09GQ.A0219114.N5aUCG.006.0656338553321.h5.mp",
-                    "key": "MOD09GQ___006/MOD/MOD09GQ.A0219114.N5aUCG.006.0656338553321.h5.mp",
-                    "path": "jk2-IngestGranuleSuccess-1558420117156-test-data/files",
-                    "size": 21708,
-                    "type": "metadata",
-                    "url_path": "{cmrMetadata.Granule.Collection.ShortName}___"
-                                "{cmrMetadata.Granule.Collection.VersionId}/"
-                                "{substring(file.name, 0, 3)}",
-                },
-                {
-                    "bucket": "cumulus-test-sandbox-protected-2",
-                    "source": "s3://cumulus-test-sandbox-protected-2/MOD09GQ___"
-                              "006/MOD/MOD09GQ.A0219114.N5aUCG.006.0656338553321.cmr.json",
-                    "key": "MOD09GQ___006/MOD/MOD09GQ.A0219114.N5aUCG.006.0656338553321.cmr.json",
-                    "fileName": "MOD09GQ.A0219114.N5aUCG.006.0656338553321.cmr.json",
-                    "type": "metadata",
-                    "url_path": "{cmrMetadata.Granule.Collection.ShortName}___"
-                                "{cmrMetadata.Granule.Collection.VersionId}/{substring(file.name, "
-                    "0, 3)}",
-                },
-            ],
             "granuleId": self.task_input_event["input"]["granules"][0]["granuleId"],
             "keys": [exp_key1, exp_key2, exp_key3],
-            "version": "006",
         }
 
         exp_grans = [exp_gran]
 
         exp_result = {"granules": exp_grans}
         self.assertEqual(exp_result, result)
 
-    @patch("extract_filepaths_for_granule.LOGGER.debug")
-    def test_task_no_granules(self, mock_debug: MagicMock):
```

Comment:
Since our input format is now just checked in `handler`, many of these tests go away.